lib/core/src/animation/animator.rs

```diff
@@ -1,22 +1,25 @@
-use super::ContinuousTimeAnimator;
-use super::FnAnimation;
+pub mod continuous;
+pub mod fixed_step;
 
+use continuous::ContinuousAnimator;
+use super::AnimationCallback;
+use crate::system::web::animation_frame_loop::AnimationFrameLoop;
 
 
 // ====================
 // === AnimatorData ===
 // ====================
 
 struct AnimatorData {
-    closure     : Box<dyn FnMut(f32)>,
+    callback    : Box<dyn FnMut(f32)>,
```

Comment:
I was writing about it in my previous review - please do not repeat yourself. You have created `AnimationCallback` type, which is defined as `FnMut(f32)`. Why are you using it sometimes and sometimes not?